### PR TITLE
Adds basic raid protection.

### PR DESCRIPTION
**Pull request by 404dcd**
This "starts the ball rolling" on issue #369 - a simple toggle command that toggles the ability of @.everyone to create instant invites, and on the ON toggle will delete all invites. It also sends a log to the logging channel. It is a root only command.

The code does work, but I'm making it a draft for now because I think we can add some extra functionality. Please voice your suggestions.

### CODE REVIEW

File: cdbot/cogs/admin.py
Comment (Sh3llcod3) [inline cdbot/cogs/admin.py:95]: But there are only two roots as of now 🤔

File: cdbot/cogs/admin.py
Comment (Sh3llcod3) [inline cdbot/cogs/admin.py:134]: Could this be placed below the first `if raidon` clause below line 127 instead? So it preps the embed and revokes the roles all under one if statement?

**Review by Sh3llcod3 — APPROVED** [review]
Good so far, maybe minor changes.

File: cdbot/cogs/admin.py
Comment (404dcd) [reply cdbot/cogs/admin.py:95]: So allow Sudos as well?

File: cdbot/cogs/admin.py
Comment (404dcd) [reply cdbot/cogs/admin.py:134]: Yep, I can do that now

**Comment by H4ckerJ4cker** [comment]
Instead of yeeting all the invites we should probs have a system in place where new members are given a default role with perms to view channels and deny it from @everyone. If raid mode is on then don't give members that role. And only show them a channel explaining what raid mode is

File: cdbot/cogs/admin.py
Comment (Sh3llcod3) [reply cdbot/cogs/admin.py:95]: Idk, you'd probably have to ask staff since that'd be their view, but I'm just saying from an availability standpoint :+1: :joy:

**Comment by H4ckerJ4cker** [comment]
Instead of yeeting all the invites maybe we can think of a system that gives users a role that has perms to view the server on join and when raid mode is on dont give that role

**Comment by H4ckerJ4cker** [comment]
What about the default invite thats mapped to that domain beano has

**Comment by 404dcd** [comment]
> What about the default invite thats mapped to that domain beano has

So, that can either be reinstated manually or we could add some functionality that hits Cloudflare's API to update it.

**Review by thebeanogamer — COMMENTED** [review]
This looks like it'll work, I'll approve it properly once I've tested it

**Review by Sh3llcod3 — APPROVED** [review]

**Review by thebeanogamer — CHANGES_REQUESTED** [review]
Ok I've now tested this. I'm happy that it works, but there's a couple small tweaks I'd like before merging.

1. Can we put the admin's name in the audit log please. I know it goes to `#log`, but having it in both would be nice
2. I'm not a massive fan of having this as a toggle. I think it would work better as the base command showing you the current value, then you having to specify off/on. The current model has too high a chance for accidently toggling it to the wrong value because you don't know the current value.

**Comment by 404dcd** [comment]
Ok, I'll get right on it.

**Review by Sh3llcod3 — APPROVED** [review]
Do it

**Review by thebeanogamer — APPROVED** [review]

**Pull request merged**
